extras/docker/fromsource/heketi-start.sh

```diff
@@ -50,12 +59,12 @@ if [[ -d "${HEKETI_PATH}" ]]; then
         sleep 1
     done
     if [[ $flock_status -ne 0 ]]; then
-        echo "Database file is read-only" | tee -a "${HEKETI_PATH}/container.log"
+        error "Database file is read-only"
```

Comment:
Is there actually an `exit` missing here?

If it is missing, then we should add it (and could actually make the `exit 1` part of the error function... Otherwise, let' maybe put `info` here?